fabric-config-api-v-2147483648/src/main/resources/fabric.mod.json

```diff
@@ -0,0 +1,28 @@
+{
+  "schemaVersion": 1,
+  "id": "fabric-config-api-v-2147483648",
+  "name": "Fabric Config API (v-2147483648)",
+  "version": "${version}",
+  "license": "Apache-2.0",
+  "icon": "assets/fabric-config-api-v-2147483648/icon.png",
```

Comment:
the icon could be a bit more, fun I think. maybe something to really make it pop

Reply:
☝️☝️☝️ I would take this advice, Pros is a graphic design genius as you can see from the Mod menu icon.